public/static/partials/user_volume_options.html

```diff
@@ -0,0 +1,25 @@
+<form name="form" novalidate ng-submit="form.$valid && createVolume()">
+  <fieldset>
+    <div style="margin-left: 10px;">
+      <div style="margin-bottom:5px">
+        <label> Volume size (GB) </label>
+        <input type="number" ng-model="createVolumeInfo.size" min="1" max="availableVolumeSize()"/>
+        <span class="warning-text" ng-show="invalidVolumeSize(createVolumeInfo.size)"><i class="fa fa-warning"></i>
+      Can have at most [[ maxVolumeSizePerUser ]] GiB per user
+        </span>
+      </div>
+      <div style="margin-bottom:5px">
+        <label> Availability Zone</label>
+        <input type="text" id="input-zone"  ng-model="createVolumeInfo.zone"/>
```

Comment:
should we validate on the client side the user has entered a valid type/zone?
Is it validated on the server side?

Reply:
I've added validation for type. Rather than add validation for zone (which may get tricky as regions are added), we can not retry CreateVolume for invalid parameter values, and this will handle zone.

Reply:
would it work to check AZs against the list of AZs in `settings.providers.aws`?